tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Command/UpToDateCommandTest.php

```diff
@@ -6,22 +6,24 @@
 use Doctrine\DBAL\Migrations\Tools\Console\Command\UpToDateCommand;
 use Doctrine\DBAL\Migrations\Tools\Console\Helper\ConfigurationHelper;
 use Doctrine\DBAL\Migrations\Version;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
 use Symfony\Component\Console\Helper\HelperSet;
 use Symfony\Component\Console\Input\ArrayInput;
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
  * @covers \Doctrine\DBAL\Migrations\Tools\Console\Command\UpToDateCommand
  */
-class UpToDateCommandTest extends \PHPUnit\Framework\TestCase
+class UpToDateCommandTest extends TestCase
 {
-    /** @var \PHPUnit_Framework_MockObject_MockObject|OutputInterface */
+    /** @var OutputInterface|MockObject */
```

Comment:
This is probably used in a bunch of other places as well, but it should probably be `OutputInterface&MockObject`.

Reply:
True, but I'm avoiding using `&` for now since it's not supported i.e. by PhpStormn at all. :( I hope it'll improve in future and PhpStorm will understand intersections.

Reply:
Makes sense - let's hope for the best. PhpStan supports it, and there's an open issue in PhpStorm for it (although relatively new, so we might have to wait a while): https://youtrack.jetbrains.com/issue/WI-39419